edb/server/protocol/auth_ext/oauth.py

```diff
@@ -28,49 +28,46 @@
 class Client:
     provider: base.BaseProvider
 
-    def __init__(self, db: Any, provider_id: str, base_url: str | None = None):
+    def __init__(
+        self,
+        db: Any,
+        provider_name: str,
+        base_url: str | None = None
+    ):
         self.db = db
         self.db_config = db.db_config
 
         http_factory = lambda *args, **kwargs: http_client.HttpClient(
             *args, edgedb_test_url=base_url, **kwargs
         )
 
-        (provider_name, client_id, client_secret) = self._get_provider_config(
-            provider_id
-        )
-
         match provider_name:
-            case "github":
+            case "builtin::oauth_github":
                 from . import github
```

Comment:
Do we need conditional imports? If it works with moving them to the top of the file please just do that.